tests/test_salesforce_lookback_window.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime, timedelta
+from tap_tester import connections, runner, menagerie
+from base import SalesforceBaseTest
+
+class SalesforceLookbackWindow(SalesforceBaseTest):
+
+    # subtract the desired amount of seconds form the date and return
+    def timedelta_formatted(self, dtime, seconds=0):
+        try:
+            date_stripped = datetime.strptime(dtime, self.START_DATE_FORMAT)
+            return_date = date_stripped + timedelta(seconds=seconds)
+
+            return datetime.strftime(return_date, self.START_DATE_FORMAT)
+
+        except ValueError:
+            try:
+                date_stripped = datetime.strptime(dtime, self.BOOKMARK_COMPARISON_FORMAT)
+                return_date = date_stripped + timedelta(seconds=seconds)
+
+                return datetime.strftime(return_date, self.BOOKMARK_COMPARISON_FORMAT)
+
+            except ValueError:
+                return Exception("Datetime object is not of the format: {}".format(self.START_DATE_FORMAT))
+
+    @staticmethod
+    def name():
+        return 'tap_tester_salesforce_lookback_window'
+
+    @staticmethod
+    def get_properties():  # pylint: disable=arguments-differ
+        return {
+            'start_date' : '2021-11-10T00:00:00Z',
+            'instance_url': 'https://singer2-dev-ed.my.salesforce.com',
+            'select_fields_by_default': 'true',
+            'api_type': 'REST',
```

Comment:
Can this test be executed against the BULK API as well?

Reply:
Updated the code to run for BULK API too.